x-pack/test/functional/page_objects/asset_details.ts

```diff
@@ -40,6 +40,13 @@ export function AssetDetailsProvider({ getService }: FtrProviderContext) {
       return container.findAllByCssSelector('[data-test-subj*="infraAssetDetailsMetricsChart"]');
     },
 
+    async getAssetDetailsNginxMetricsCharts() {
+      const container = await testSubjects.find('infraAssetDetailsNginxMetricsChartGrid');
+      return container.findAllByCssSelector(
```

Comment:
Can we use `container.findByTestSubject` here? Would be a bit cleaner.

Reply:
`findByTestSubject` will return 1 item with a test id equal to the value passed, here I want to get all items with the test id starting with infraAssetDetailsNginxMetricsChart so I use `data-test-subj*=` to get them, not sure if there is a better way to do that 🤔